gradle-plugin/build.gradle

```diff
@@ -1,4 +1,9 @@
-description 'neow3j: Gradle Plugin'
+plugins {
+    id 'java-gradle-plugin'
+    id 'com.gradle.plugin-publish' version '1.2.1'
+}
+
+description = "Gradle plugin to compile Java smart contracts to NeoVM"
```

Comment:
```suggestion
description = "neow3j: Gradle plugin to compile Java smart contracts to NeoVM"
```

We should keep "neow3j: " as prefix, since all the packages have this:

![image](https://github.com/neow3j/neow3j/assets/2124276/aa721fbd-11d9-4e41-9a27-10f576f9a5aa)